theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer-contribution.ts

```diff
@@ -178,6 +186,10 @@ export class TraceViewerContribution extends WidgetOpenHandler<TraceViewerWidget
         registry.registerCommand(OpenTraceCommand, {
             execute: () => this.launchTraceServer()
         });
+        registry.registerCommand(OpenTraceWithPathCommand, {
+            isVisible: () => false,
+            execute: (path: string, options: TraceViewerWidgetOpenerOptions) => path && this.open(new URI(path), options),
```

Comment:
Do you need the TraceViewerWidgetOpenerOptions? The options has the traceUUID which is used when opening a trace that is already on the server and the traceUUID is known. That is triggered from the `Opened Traces` view. If the option is passed and the traceUUID is set then the path is ignored.

Reply:
I don't need it - I included it as I tried to follow the existing guidance. Basically I though that the plan was to keep extending the options as additional flags were needed to control the open.

But as that can be added later once there are additional options, I will update the patch without this part.